config/analytics.yml

```diff
@@ -9,6 +10,19 @@ shared:
     - created_at
     - updated_at
     - vacancy_id
+  employments:
+    - id
+    - organisation
+    - job_title
+    - salary
+    - subjects
+    - current_role
+    - main_duties
+    - started_on
+    - ended_on
+    - job_application_id
+    - created_at
+    - updated_at
   feedbacks:
```

Comment:
@david-mears-dfe are there any new fields in feedbacks introduced along with applications?

Reply:
There is, but we already accounted for it: 'application_id'.